Sources/Afterpay/BadgeView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  BadgeView.swift
+//  Afterpay
+//
+//  Created by Adam Campbell on 31/7/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public final class BadgeView: UIView {
+
+  public enum Style {
+    case whiteOnBlack, blackOnWhite
+
+    var svg: SVG {
+      switch self {
+      case .whiteOnBlack:
+        return .badgeWhiteOnBlack
+      case .blackOnWhite:
+        return .badgeBlackOnWhite
+      }
+    }
+  }
+
+  private var style: Style = .whiteOnBlack
```

Comment:
Is this something we're planning to expose outside of the SDK?

Reply:
It probably will end up being exposed. At the moment it's only configured via the init.